.github/workflows/ephemeral-instances-pr-comment.yml

```diff
@@ -52,21 +52,26 @@ jobs:
           token: ${{ steps.generate_token.outputs.token }}
           ref: main
           path: ephemeral
-
+      - name: Get latest grafana version number
+        run: |
+          # if package.json contains e.g. 11.0.0-pre, this writes 11.0.0 to version.txt
+          curl https://raw.githubusercontent.com/grafana/grafana/main/package.json | jq -r .version | grep -o  '^[0-9\.]*' > version.txt
```

Comment:
No need to `curl` here, since we're in the right repo - we should be able to just `cat ./package.json | ...`

Reply:
Oh i didn't see it check out the code in the action. Is that implicit? I'll try it.

Reply:
hm, trying it in [this action,](https://github.com/grafana/grafana/actions/runs/8423465198/job/23065160826) i see


no such file or directory. I don't think we clone the repo in this particular action.

Reply:
I added `ls -la $GITHUB_WORKSPACE` to [this workflow](https://github.com/grafana/grafana/actions/runs/8423524273/job/23065367969) run and got

```
 drwxr-xr-x 3 runner runner 4096 Mar 25 16:28 .
drwxr-xr-x 3 runner runner 4096 Mar 25 16:28 ..
drwxr-xr-x 7 runner runner 4096 Mar 25 16:28 ephemeral
```

So I'm pretty sure we don't clone grafana/grafana, and I'd prefer just to download the one file in this workflow 👍

Reply:
OH - I misread the above action, I thought it was checking out from this repo 🤦‍♂️